### DISCUSSION TITLE
Is the data updated for this 2023 season?

### DISCUSSION THREAD

**Discussion by SQL499** [kyleskom/NBA-Machine-Learning-Sports-Betting#155] [General]
I have seen the Dataset files, and I have found a file called "DataSet-2022-23.xlsx" when I opened it, it had all the data of the matches, but I have reviewed the file well, and they only have the matches up to date 2022 -06-16, so that worries me since if you only have data up to that date, the predictions could be less accurate and I even dare to say that the predictions would no longer be valid, would they?

**Comment by nealmick** [comment]
I don't think the NBA game changes much in the course of a few months, or even a few years... teams might get better or worse, but this is reflected by updated stats.  The essence of the game might slowly change, not quickly.... 3 point shots have become more valuable over the last 5 years...  but if you look at a game 2 years ago, compared to a game yesterday, it seems very similar to me.  Also one of the main things to look at when evaluating a model is how well it generalizes to test data never seen before, and not overfitting.  A model that cant generalize to future games, is kinda missing the point....  I have also specifically tested this theory, by stopping training up to a full season early, then using the last 250 games or so as testing, leaving over 1000 game gap between training and testing, It isn't detrimental and docent destroy accuracy on its own.  Its more harmful to just not have those games or 1000 less games to train on, but is negligible if you added another 1000 games to the beginning of the dataset.

**Reply by SQL499** [reply]
Of course I can understand that point of view, in fact I even agree with you, the nba in general has not changed much compared to last year or even 2021, but I suppose that only says that it does not harm the predictions that the team will go to. be the winner right? Because I feel that something that always changes in the nba is the average number of points that there will be in the game taking into account the 2 teams that are going to play, mainly for that reason I have asked what I asked, not having the most current data to date Wouldn't it affect the under/over point predictions in the match being played?

**Reply by kyleskom** [reply]
It does pull the current teams data evey day to use for predictions. What isn't updated daily is the data used for model training which shouldn't change that much over short periods of time.

**Reply by jidemaniax** [reply]
Please i have been wondering how to train the model with new data further down the line. Also big thank you just going the folders and the notes has helped learn new things.

**Reply by jidemaniax** [reply]
Hey man it would be amazing if you could make a patreon. I will gladly pay for a tutorial. I just wasted days struggling with youtube videos to create something.

**Comment by Marcfeitosa** [comment]
I have the same question
    Em sexta-feira, 24 de março de 2023 às 08:46:35 BRT, jidemaniax ***@***.***> escreveu:  
 
 


Please i have been wondering how to train the model with new data further down the line
